frontend_tests/node_tests/markdown.js

```diff
@@ -861,7 +859,10 @@ test("katex_throws_unexpected_exceptions", () => {
             },
         },
         () => {
-            markdown.apply_markdown(message);
+            assert.throws(() => markdown.apply_markdown(message), {
+                name: "Error",
+                message: "some-exception\nPlease report this to https://github.com/chjj/marked.",
```

Comment:
It's a bit funny that this suggests a report to upstream; we may want to add a bonus commit that replace that URL with `github.com/zulip/zulip` given how much we've forked `marked` at this point.